src/rez/pip.py

```diff
@@ -134,11 +132,19 @@ def find_python_in_context(context):
     # Create a copy of the context with systems paths removed, so we don't
     # accidentally find a system python install.
     #
+    success = (context.status == ResolverStatus.solved)
+    if not success:
+        context.print_info(buf=sys.stderr)
```

Comment:
Should we add a custom error message too? Something like "Failed to resolve ... "